docs/source/conf.py

```diff
@@ -0,0 +1,42 @@
+# Configuration file for the Sphinx documentation builder.
+
+# -- Project information
+
+import sys
+from pathlib import Path
+
+project_root = str(Path(__file__).parent.parent.parent)
+
+sys.path.insert(0, project_root)
+
+with open(Path.cwd().parent.parent / "oml" / "__init__.py", "r") as f:
+    version = f.read().split('"')[-2]
+
+project = "Open Metric Learning"
+author = "Shabanov Aleksei"
+
+# -- General configuration
+
+extensions = [
+    "sphinx.ext.duration",
+    "sphinx.ext.doctest",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.napoleon",
+]
+
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3/", None),
+    "sphinx": ("https://www.sphinx-doc.org/en/master/", None),
+}
+intersphinx_disabled_domains = ["std"]
+
+templates_path = ["_templates"]
+
+# -- Options for HTML output
+
+html_theme = "sphinx_rtd_theme"
+
+# -- Options for EPUB output
+epub_show_urls = "footnote"
```

Comment:
rm?
we probably will never build docs in .epub

Reply:
ok